src/libcmd/command.hh

```diff
@@ -22,7 +22,7 @@ static constexpr Command::Category catSecondary = 100;
 static constexpr Command::Category catUtility = 101;
 static constexpr Command::Category catNixInstallation = 102;
 
-static constexpr auto installablesCategory = "Options that change the interpretation of installables";
+static constexpr auto installablesCategory = "Options that change the interpretation of [installables](../../glossary.md#gloss-installable)";
```

Comment:
Instead of linking to the glossary, it would be better to link to the "Installables" section of the `nix` manpage, which is much more complete.

Reply:
Shouldn't everything in the man page be also in the manual? I would think the man page info can go in the manual, and the glossary can be a 1-sentance thing that points to it.

I am not sure which we should link, but that is a general issue when the same info (once we do the above!) is available two different ways.

Reply:
@edolstra the idea was to link to an at-a-glance overview of what the term means. The full page requires scrolling just to see all the options.

But sure, I can also add an overview on the full page.

Reply:
@Ericson2314 The man page is part of the manual (https://nixos.org/manual/nix/stable/command-ref/new-cli/nix.html#installables) so it's fine to link to that.

@fricklerhandwerk The at-a-glance overview should ideally (also) be part of the installables section of the `nix` man page, since that's what most users will see (e.g. when they run `nix help`).

Reply:
Sounds good. I was sort of thinking the regular manual info shouldn't need to refer to the man pages (even though the regular manual does contain the man page) but then it turned out that the updated sections were all man pages anyways!